elements/pfe-clipboard/src/pfe-clipboard.scss

```diff
@@ -74,16 +107,14 @@ $LOCAL-VARIABLES: (
   width: 100%;
 }
 
-// Note: Focus states need to be defined before hover states
-:host(:not([aria-disabled="true"]):focus),
-:host(:not([aria-disabled="true"]).focus-within) {
-  --pfe-clipboard--Color: #{pfe-local(Color--focus)};
+:host([on="dark"]) {
+  --pfe-clipboard--icon--Color: #{pfe-local(icon--Color--dark)};
+  --pfe-clipboard--icon--Color--hover: #{pfe-local(icon--Color--hover--dark)};
 }
 
-// Note: Hover states need to be defined after focus states
-:host(:not([aria-disabled="true"]):hover),
-:host(:not([aria-disabled="true"])) ::slotted(:hover) {
-  --pfe-clipboard--Color: #{pfe-local(Color--hover)};
+:host([on="saturated"]) {
+  --pfe-clipboard--icon--Color: #{pfe-local(icon--Color--saturated)};
+  --pfe-clipboard--icon--Color--hover: #{pfe-local(icon--Color--hover--saturated)};
```

Comment:
Explicitly adding theme context hooks for icon color.  We don't have to include text color variables because that's automatically covered by broadcast variables.